### ISSUE TITLE
KeyError: 'y' in train_risk_slim.py

### ISSUE THREAD

**Issue by nathandxn** [ustunb/riskslim#14]
There is an issue with the y = data['y'] definition

![image](https://user-images.githubusercontent.com/43524608/117594445-a408f200-b103-11eb-8f7c-ab5d397fc6fe.png)

**Comment by nickalewis** [comment]
Hey, nathandxn, I think the issue might be that the y should be capitalized. So instead of 'y', it should be 'Y'

**Comment by nathandxn** [comment]
@nickalewis correct, I made that change on my end to get it working. I made a PR to update in the project: https://github.com/ustunb/risk-slim/pull/15
